tests/users/test_api_list_users.py

```diff
@@ -16,7 +16,7 @@ def setUp(self):
         super(TestListUsersApi, self).setUp()
 
         self.verified_user = UserModel(
-            name=user1['name'],
+            name=user1['name'] + '    Example',
```

Comment:
Why Example?

Reply:
I've added this so that the verified user would contain spaces in her/his name and added a test for this as well. :)
`test_list_users_api_with_a_search_query_with_spaces_resource_auth`